examples/named_cluster/main.tf

```diff
@@ -77,4 +77,9 @@ module "aks_cluster_name" {
   rbac_aad                          = true
   rbac_aad_managed                  = true
   role_based_access_control_enabled = true
+
+  # KMS encrption
+  key_vault_kms_enabled = true
```

Comment:
These variable name is invalid. Should be just `kms_enabled`

Reply:
Yea. Sorry, copy pasta error before refactoring :D